rust-toolchain.toml

```
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly"
+components = [ "rustfmt", "clippy" ]
+profile = "default"
```

Sorry for nitpicking but I don't think we should specify `profile` if it's just `default`, also TOML formatting is unusual.

```suggestion
components = ["rustfmt", "clippy"]
```

---

haha, no worries :D